README.md

```diff
@@ -270,13 +270,36 @@ Multiple `--format` options can be specified as long as they all have distinct o
 
 Specifies options to pass through to the actual test-runner. In the default template (and most commonly across modules), this is [rspec](https://relishapp.com/rspec/rspec-core/docs/command-line).
 
+### `pdk bundle` command
+
+This experimental command allows you to execute arbitrary commands in a bundler context within the module you're currently working on. Arguments to this command are passed straight through to bundler.
+
+Note that for most advanced uses it is required to use the `--` to separate bundler options from pdk options. Compare the following two commands:
```

Comment:
The command text says this is for 'advanced users' - we might want at least that level of warning here; if not a stronger warning that the user should know what they're doing (e.g. they might install a gem to system with this command, causing problems on their machine)?